bindings/c/test/mako/utils.c

```diff
@@ -80,6 +80,27 @@ void genkey(char* str, char* prefix, int prefixlen, int prefixpadding, int num,
 	str[len - 1] = '\0';
 }
 
+void update_key_prefix(char* prefix, int prefixlen, int pos) {
+	memcpy(prefix, KEYPREFIX, KEYPREFIXLEN);
+	if (pos < prefixlen) {
+		randnumstr(prefix + pos, prefixlen - pos);
+	}
+}
+
+void update_key_prefix_with_timestamp(char* prefix, int prefixlen, struct timespec* cur_time) {
+	long mask = 1;
+	while (mask <= cur_time->tv_nsec) {
+		mask *= 10;
+	}
+	mask /= 10;
```

Comment:
We shouldn't ignore leading zeros. e.g. tv_nsec=002418523, then we should start with 002

Reply:
Yeah, that's more reasonable. Thanks for the notification!